sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/simpleCosting.scala

```diff
@@ -35,15 +36,48 @@ case class SimpleCost(value: Long) extends Cost {
 }
 
 /**
- * A simple implementation of [[CostEvaluator]], which counts the number of
- * [[ShuffleExchangeLike]] nodes in the plan.
+ * A skew join aware implementation of [[Cost]], which consider shuffle number and skew join number
```

Comment:
Can we add more description on how the cost is calculated in the presence of skew joins? What happens if there's two or more extra shuffles by adding a skew join optimization?

Reply:
yea, add more comment to show how we pick the cost with skew join and shuffle.